lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Dictionary.java

```diff
@@ -16,14 +16,7 @@
  */
 package org.apache.lucene.analysis.hunspell;
 
-import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.LineNumberReader;
-import java.io.OutputStream;
+import java.io.*;
```

Comment:
Did you run gradlew tidy? Wildcard imports shouldn't be there, hence the question.

Reply:
The imports were collapsed by the IDE, but `gradle precheck` didn't complain of wildcards. I can expand them back if that matters.

Reply:
Darn. I see you're right. They should be expanded - enforcement of this has been lost somewhere during ant-gradle migration. gjf doesn't deal with this [1].

You can also leave it in if you wish - automatic import expansion/ ordering can be done later (please feel free to file a separate issue!).

https://github.com/google/google-java-format/issues/113